css/style.css

```diff
@@ -0,0 +1,20 @@
+html{
+  font-size : 20px;
+}
+
```

Comment:
Please use HTML class(s) and id(s) when styling elements and avoid using the HTML tag (use the universal selector * if necessary )